src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -0,0 +1,92 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.Conversation;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.data.Message;
+import codeu.model.store.basic.MessageStore;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.mindrot.jbcrypt.BCrypt;
+
+/** Servlet class responsible for the Admin page. */
+public class AdminServlet extends HttpServlet {
+
+  private UserStore userStore;
+  private ConversationStore ConversationStore;
```

Comment:
The instance name should have camelCase, and it should be different from the class name. It's interesting that the Java compiler didn't complain about this. :)